packages/web/src/components/notification/Notifications/RemixCreateNotification.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+
+import { RemixCreate } from 'common/store/notifications/types'
+
+import { EntityLink } from './EntityLink'
+import { NotificationBody } from './NotificationBody'
+import { NotificationFooter } from './NotificationFooter'
+import { NotificationHeader } from './NotificationHeader'
+import { NotificationTile } from './NotificationTile'
+import { NotificationTitle } from './NotificationTitle'
+import { TwitterShareButton } from './TwitterShareButton'
+import { UserNameLink } from './UserNameLink'
+import { IconRemix } from './icons'
+import { TrackEntity } from './types'
+
+const messages = {
+  title: 'New remix of your track',
+  by: 'by'
+}
+
+type RemixCreateNotificationProps = {
+  notification: RemixCreate
+}
+
+export const RemixCreateNotification = (
+  props: RemixCreateNotificationProps
+) => {
+  const { notification } = props
+  const { user, entities, entityType, timeLabel, isRead } = notification
+
+  const entity = entities.find(
+    track => track.track_id === notification.childTrackId
+  ) as TrackEntity
+
+  return (
+    <NotificationTile notification={notification}>
+      <NotificationHeader icon={<IconRemix />}>
+        <NotificationTitle>{messages.title}</NotificationTitle>
+      </NotificationHeader>
+      <NotificationBody>
+        <EntityLink entity={entity} entityType={entityType} />
+        <span>{messages.by}</span>
+        <UserNameLink user={user} notification={notification} leftMargin />
```

Comment:
Feels like the previous span should have a `margin-inline-end` vs having a prop for left margin in `UserNameLink`?

Reply:
completely agree, was wigging out on how to do this, styling def aint my strongsuit